sdk/Cargo.toml

```diff
@@ -221,30 +221,15 @@ path = "examples/how_tos/accounts_and_addresses/create_mnemonic.rs"
 required-features = ["client"]
 
 [[example]]
-name = "create_account"
-path = "examples/how_tos/accounts_and_addresses/create_account.rs"
-required-features = ["rocksdb", "stronghold"]
-
-[[example]]
-name = "list_accounts"
-path = "examples/how_tos/accounts_and_addresses/list_accounts.rs"
+name = "create_wallet"
+path = "examples/how_tos/accounts_and_addresses/create_wallet.rs"
 required-features = ["rocksdb", "stronghold"]
 
 [[example]]
 name = "check_balance"
 path = "examples/how_tos/accounts_and_addresses/check_balance.rs"
 required-features = ["rocksdb", "stronghold"]
 
-[[example]]
-name = "list_addresses"
-path = "examples/how_tos/accounts_and_addresses/list_addresses.rs"
-required-features = ["rocksdb", "stronghold"]
-
-[[example]]
-name = "create_address"
-path = "examples/how_tos/accounts_and_addresses/create_address.rs"
```

Comment:
Also remove the files of the disabled examples ?

Reply:
probably re-inserted during merge. thanks

Reply:
Done